Shared/ViewModels/UserSignInViewModel.swift

```diff
@@ -215,7 +181,7 @@ final class UserSignInViewModel: ViewModel, Eventful, Stateful {
         }
     }
 
-    private func signIn(username: String, password: String, policy: UserAccessPolicy) async throws -> UserState {
+    private func makeSignInRequest(username: String, password: String) async throws -> AuthenticationResult {
```

Comment:
Don't need a separate function for this, just inline in the one place it's used. Also, this would be returning the result, not the actual request.